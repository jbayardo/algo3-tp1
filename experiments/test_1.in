2
1
30
3 5 7 8 10 12 25 29
50
1 3 5 7 9 20 34
80
4 8 9 12 40 45 50 62 77 80
4
5 10 15 20 25
8
1 10 30 39 51 67
10
2 15 38 49 65 89
10
50 55 59 60 61 63 65 70 80
8
3 8 12 20 24 36 40 47 50
30
12 36 40 50 57 60 62 63 68 70 75 80