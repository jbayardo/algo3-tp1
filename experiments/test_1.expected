1
8
7
10
0
0
0
6
2
9